src/pages/Survey/components/SurveyResult/SurveyResult.js

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
+import './SurveyResult.scss';
+
+const SurveyResult = () => {
+  const [perfumeList, setPerfumeList] = useState([]);
+
+  const location = useLocation();
+
+  const decodeUri = decodeURI(location.search);
+
+  const navigate = useNavigate();
+
+  const userName = () => {
+    const nameStart = decodeUri.indexOf(`=`);
+    const nameEnd = decodeUri.indexOf('&');
+    const name = decodeUri.slice(nameStart + 1, nameEnd);
+    let result;
+    if (name.length < 2) {
+      return (result = '회원');
+    } else result = name;
+    return result;
+  };
+
+  function randomNumberInRange(min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+
+  const randomNumber = randomNumberInRange(1, 16);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/perfume_list.json')
```

Comment:
```suggestion
    fetch('data/perfume_list.json')
```
절대경로로 접근할 수 있고
http://localhost:3000 은 생략 가능합니다. 포트 번호가 달라지면(ex. localhost:3001) 코드 전체를 바꿔야 하기 때문에 수정해주시기 바랍니다. 아래 코드 참고해주세요.
```javascript
const foo = () => {
	// fetch("http://localhost:3000/images/1")
	fetch("/images/1")
}

```